site/content/docs/user/configuration.md

```diff
@@ -42,6 +42,20 @@ The following high level options are available.
 
 NOTE: not all options are documented yet!  We will fix this with time, PRs welcome!
 
+### Name Your Cluster
+
+You can give your cluster a name by specifying it in your config:
+
+{{< codeFromInline lang="yaml" >}}
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+name: app-1-cluster
+{{< /codeFromInline >}}
+
+This name will be used unless you invoke `kind create cluster` with the `--name`
+option as well (for example `kind create cluster --config=config.yml --name other`),
+in which case `other` will be used since CLI args take precedence.
```

Comment:
I think this is generally true for every parameter. i.e. CLI flags will overwrite whatever is specific in the config file.
Since this page specifically deals with the config, I think just the config example should suffice here in this subsection.

Even better if we can move this note as a general note at the top somewhere :)

Reply:
+1 for a general note and also the CLI usage documenting this.

Reply:
@amwat @BenTheElder thanks for the feedback! I pulled that CLI precedence note up and trimmed down the `name` section specifically. Good call out!